src/lib/mailer/index.js

```diff
@@ -0,0 +1,201 @@
+import { createTransport } from 'nodemailer';
+import Email from 'email-templates';
+import * as path from 'path';
+import { logger } from '../../logger';
+
+const {
+  SMTP_HOST, SMTP_PORT, SMTP_USER, SMTP_PASS, SMTP_SECURE, SEND_NON_PRODUCTION_NOTIFICATIONS,
+} = process.env;
+
+// nodemailer expects this value as a boolean.
+const secure = SMTP_SECURE !== 'false';
+
+const defaultTransport = createTransport({
+  host: SMTP_HOST,
+  port: SMTP_PORT,
+  secure,
+  auth: {
+    user: SMTP_USER,
+    pass: SMTP_PASS,
+  },
+});
+
+const send = SEND_NON_PRODUCTION_NOTIFICATIONS === 'true';
```

Comment:
I _think_ this will actually end up preventing emails being sent in production (once `SEND_NOTIFICATIONS` is true there) by forcing `send` to false when we really want to either have it be true or not be set at all.

Reply:
Ah, you are correct. I will fix that.